Edit omega.generate_conformers to resolve #97

davidlmobley opened the pull request:
Modifying generate_conformers to default to using loose MMFF atom
typing (to allow coverage of a wider range of molecules)

kyleabeauchamp commented on gaff2xml/openeye.py:
```diff
@@ -154,7 +154,7 @@ def smiles_to_oemol(smiles):
 
-def generate_conformers(molecule, max_confs=800, strictStereo=True, ewindow=15.0, rms_threshold=1.0):
+def generate_conformers(molecule, max_confs=800, strictStereo=True, ewindow=15.0, rms_threshold=1.0, stricTypes = False):
     """Generate conformations for the supplied molecule
```
I think you have a typo: `stricTypes` should be `strictTypes`, right?

kyleabeauchamp commented:
PS: FYI the Travis online tests are unable to test any OpenEye functionality, so you have to test those manually on your local install.

davidlmobley commented:
Indeed, fixing/testing.

davidlmobley commented:
So I fixed the typo and tested this, but in the process found that this does NOT fix the issue with my particular molecule because there is apparently no MMFF parameter for any type of boron in this environment. On further review of my notes, I am unable to find other good examples where this has posed major problems (though I've certainly seen a couple over the years) leading me to conclude the issue is not as widespread as I first thought. Still, I expect sooner or later a student will need to override the strictTyping option.

So, I updated the pull request to:
- fix the typo
- switch the default behavior back to what it was originally (using strict typing) and only optionally allow an override

kyleabeauchamp commented:
Looks good to me
